x-pack/plugins/apm/public/components/fleet_integration/apm_custom_assets_extension.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  CustomAssetsAccordionProps,
+  CustomAssetsAccordion,
+} from '../../../../fleet/public';
+import { useKibana } from '../../../../../../src/plugins/kibana_react/public';
+import { ApmPluginStartDeps } from '../../plugin';
+
+export function ApmCustomAssetsExtension() {
+  const { http } = useKibana<ApmPluginStartDeps>().services;
+  const basePath = http?.basePath.get();
+
+  const views: CustomAssetsAccordionProps['views'] = [
+    {
+      name: 'Services',
+      url: `${basePath}/app/apm`,
```

Comment:
This is causing a full page refresh when link to it, I think we can solve this by using `RedirectAppLinks` and add an app property here https://www.elastic.co/guide/en/kibana/master/kibana-navigation.html#navigating-between-kibana-apps 

Also I think we can use the `PLUGIN_ID` constant for apm here no?

Reply:
My mistake - I misunderstood the original comment. These should be able to all be swapped out to `PLUGIN_ID` since they link into each respective app.

Reply:
Updated these in endpoint + synthetics, but APM doesn't actually have a constant defined for the app id, and I don't think that it's worth adding scope to update that plugin's registration/start-up process to use a `PLUGIN_ID` or `APP_ID` constant. 

If this is something that we want to capture separately I can file an issue, but I think it'd need input from the folks on the APM team.